### PR TITLE
Added L2 changes of esignet plugin 

### PR DESCRIPTION
**Pull request by kaifk468**

### CODE REVIEW

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaError.java
Comment (ase-101) [inline mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaError.java:1]: Why is this required?

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaResponse.java
Comment (ase-101) [inline mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaResponse.java:1]: Why is this required?

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/GoogleRecaptchaValidatorService.java
Comment (ase-101) [inline mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/GoogleRecaptchaValidatorService.java:1]: Why is this required?

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/LoggerAuditService.java
Comment (ase-101) [inline mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/LoggerAuditService.java:1]: Why is this required?

File: mock-plugin/pom.xml
Comment (ase-101) [inline mock-plugin/pom.xml:1]: What is the change in this file?

Comment (kaifk468) [reply mock-plugin/pom.xml:1]: Nothing just some spaces got changed

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaError.java
Comment (kaifk468) [reply mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaError.java:1]: deleted

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaResponse.java
Comment (kaifk468) [reply mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/dto/ReCaptchaResponse.java:1]: deleted

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/GoogleRecaptchaValidatorService.java
Comment (kaifk468) [reply mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/GoogleRecaptchaValidatorService.java:1]: deleted

File: mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/LoggerAuditService.java
Comment (kaifk468) [reply mock-plugin/src/main/java/io/mosip/esignet/plugin/mock/service/LoggerAuditService.java:1]: deleted

**Review by ase-101 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Review by ase-101 — APPROVED** [review]
